---
title: Kraken taxonomy Workflow landing page
type: guide
contributors: Valentine Murigneux, Mike Thang
description: How-to guide for Kraken Taxonomy classification workflow on Galaxy Australia.
affiliations: QCIF
toc: true
sidebar: false
---


## About 

This document describes how to use a shotgun metagenomics workflow on Galaxy Australia. The aim of this workflow is to handle the routine part of shotgun metagenomics data processing. The workflow is using the tools Kraken2 and Bracken for taxonomy classification and the KrakenTools to evaluate diversity metrics.

This workflow represents just one way of processing data for a ‘typical’ shotgun metagenomics experiment – there are many other options!  


## Please cite this guide as follows




## Acknowledgements

This guide makes use of the ELIXIR toolkit theme: [![theme badge](https://img.shields.io/badge/ELIXIR%20toolkit%20theme-jekyll-blue?color=0d6efd)](https://github.com/ELIXIR-Belgium/elixir-toolkit-theme)


## References

This workflow depends on the following tools and resources:
* [Kraken2](https://github.com/DerrickWood/kraken2/blob/master/LICENSE)
* [Bracken](https://github.com/jenniferlu717/Bracken/blob/master/LICENSE)
* [KrakenTools](https://github.com/jenniferlu717/KrakenTools/blob/master/LICENSE)
* Krona: [publication](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-385)
* Galaxy: [Galaxy project](https://galaxyproject.org/) 
* Galaxy Australia: [Galaxy Australia service](https://usegalaxy.org.au/)

